docs/development/adding-a-package-manager.md

```diff
@@ -1,16 +1,16 @@
 # Adding a Package Manager
 
-This document describes the steps to take if you are interested in adding new language/package manager support.
+This document describes the steps to take if you want to add a new language/package manager.
 
-### Code structure
+## Code structure
 
-Each package manager lives under `lib/manager/*`, and are often tightly coupled to datasources under `lib/datasource/*`.
+Each package manager lives under `lib/manager/*`, and is often tightly coupled to datasources under `lib/datasource/*`.
 
-Versioning logic (e.g. semver, pep440) lives under `lib/versioning/*`.
+Versioning logic (e.g. SemVer, PEP 440) lives under `lib/versioning/*`.
```

Comment:
Do we want to standardize on a standard phrase?

I've seen at least these variations in our docs:

- e.g. SemVer, PEP 440
- for example, SemVer, PEP 440
- .i.e. SemVer, PEP 440

What do you like best?

Reply:
We use lower case like `semver` and `pep440` as identifiers, but they should be backtick-escaped if so. Otherwise capitalize according to the style guide of whichever external party they belong to. "e.g." is probably best in most situations